src/Trakx.CoinGecko.ApiClient.Tests/Trakx.CoinGecko.ApiClient.Tests.csproj

```diff
@@ -33,8 +33,8 @@
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
       <PrivateAssets>all</PrivateAssets>
     </PackageReference>
-    <PackageReference Include="Trakx.Common.Testing" Version="1.1.5" />
-    <PackageReference Include="Trakx.Common.Testing.Documentation" Version="1.1.5" />
+    <PackageReference Include="Trakx.Common.Testing" Version="1.1.8" />
+    <PackageReference Include="Trakx.Common.Testing.Documentation" Version="1.1.8" />
```

Comment:
You can keep only the Trakx.Testing.Documentation package, since the testing one is installed automatically by it

Reply:
OK thanks